lib/data_update_scripts/20201217184442_append_community_to_community_name.rb

```diff
@@ -0,0 +1,9 @@
+module DataUpdateScripts
+  class AppendCommunityToCommunityName
+    def run
+      SiteConfig.where(var: "community_name").find_each do |community_name|
+        community_name.update!(value: "#{community_name.value} Community")
+      end
+    end
+  end
+end
```

Comment:
I added this `data_update` script that appends "Community" to `community_name` per your suggestions @vaidehijoshi and @benhalpern . How do we feel about this? I tested it out locally and it worked as expected (see screenshot below for an example of the `community_name` with the appended "Community" after running the `data_update` script locally)!
<img width="788" alt="Screen Shot 2020-12-17 at 1 45 45 PM" src="https://user-images.githubusercontent.com/32834804/102541846-2c5ffb00-406e-11eb-8c17-8948f165ef78.png">

Reply:
Question: if we decide to go with this `data_update` script in this PR, will it be okay to keep _both_ `data_update` scripts in this PR or will it be necessary to split the two scripts up between this PR and another PR?